Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/TapToPlace.cs

```diff
@@ -0,0 +1,373 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Physics;
+using Microsoft.MixedReality.Toolkit.Utilities.Solvers;
+using System.Runtime.CompilerServices;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.Serialization;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor.Solvers
+{
+    /// <summary>
+    /// Tap to place is a far interaction component used to place objects on a surface.
+    /// </summary>
+    public class TapToPlace : Solver, IMixedRealityPointerHandler
+    {
+        [Space(10)]
+        [SerializeField]
+        [Tooltip("If true, the game object to place will start selected.  The object will immediately start" +
+" following the TrackedTargetType (Head or Controller Ray) and then a tap is required to place the object." +
+" This value must be modified before Start() is invoked in order to have any effect")]
```

Comment:
Nit: Indent this so that it's like the other ones below (i.e. I think 16 spaces? Or something else greater than zero!)

Reply:
Fixed